### ISSUE TITLE
errro when getting channel

### ISSUE THREAD

**Issue by synhya1** [SuspiciousLookingOwl/youtubei#115]
TypeError: Cannot read properties of undefined (reading 'browseEndpoint')
    at VideoCompactParser.loadVideoCompact (C:\Users\somes\OneDrive\바탕 화면\music\creativewave_crawler\node_modules\.pnpm\youtubei@1.5.4\node_modules\youtubei\dist\cjs\youtube\VideoCompact\VideoCompactParser.js:27:18)
    at VideoCompact.load (C:\Users\somes\OneDrive\바탕 화면\music\creativewave_crawler\node_modules\.pnpm\youtubei@1.5.4\node_modules\youtubei\dist\cjs\youtube\VideoCompact\VideoCompact.js:32:49)
    at C:\Users\somes\OneDrive\바탕 화면\music\creativewave_crawler\node_modules\.pnpm\youtubei@1.5.4\node_modules\youtubei\dist\cjs\youtube\Channel\ChannelParser.js:61:87
    at Array.map (<anonymous>)
    at ChannelParser.parseShelves (C:\Users\somes\OneDrive\바탕 화면\music\creativewave_crawler\node_modules\.pnpm\youtubei@1.5.4\node_modules\youtubei\dist\cjs\youtube\Channel\ChannelParser.js:59:18)    
    at ChannelParser.loadChannel (C:\Users\somes\OneDrive\바탕 화면\music\creativewave_crawler\node_modules\.pnpm\youtubei@1.5.4\node_modules\youtubei\dist\cjs\youtube\Channel\ChannelParser.js:43:40)     
    at Channel.load (C:\Users\somes\OneDrive\바탕 화면\music\creativewave_crawler\node_modules\.pnpm\youtubei@1.5.4\node_modules\youtubei\dist\cjs\youtube\Channel\Channel.js:20:39)
    at Client.<anonymous> (C:\Users\somes\OneDrive\바탕 화면\music\creativewave_crawler\node_modules\.pnpm\youtubei@1.5.4\node_modules\youtubei\dist\cjs\youtube\Client\Client.js:103:60)
    at Generator.next (<anonymous>)
    at fulfilled (C:\Users\somes\OneDrive\바탕 화면\music\creativewave_crawler\node_modules\.pnpm\youtubei@1.5.4\node_modules\youtubei\dist\cjs\youtube\Client\Client.js:5:58)

channelId: UCAsvCHa0oOQrdw1kUU9YuLQ
code: executed: await youtube.getChannel(channelId);

**Comment by ShinyRa** [comment]
I replicated the same error by using
```javascript
const channel = await youtube.getChannel("UCAsvCHa0oOQrdw1kUU9YuLQ");
```
Though, I was able to extract the channel data successfully using this approach.
```javascript
 const channel = await youtube.findOne("UCAsvCHa0oOQrdw1kUU9YuLQ", {type: "channel"}); 
```

**Comment by SuspiciousLookingOwl** [comment]
Fixed on `1.6.3`
